.travis.yml

```diff
@@ -0,0 +1,20 @@
+sudo: false
+language: cpp
+
+matrix:
+  include:
+    - os: linux
+      dist: trusty
+      compiler: gcc
```

Comment:
Can we add clang as well while we're here?

Reply:
Also, what standard library is this using?

Reply:
Currently clang is not working since clang seems to use gcc system include
I'm currently investigating. Let's start little then add it later...

Reply:
Concerning standard lib, I don't get it.
I means usually gcc and standard lib are shipped together on GNU/distro
So since linux Trusty image is shipped with gcc 4.8.2 you'll get standard lib "4.8.2"
cf https://gcc.gnu.org/onlinedocs/libstdc++/manual/api.html#api.rel_480